setup.py

```diff
@@ -1,60 +0,0 @@
-"""A setuptools based setup module for dowhy.
-
-Adapted from:
-https://packaging.python.org/en/latest/distributing.html
-https://github.com/pypa/sampleproject
-"""
-
-from setuptools import setup, find_packages
-from codecs import open
-from os import path
-
-here = path.abspath(path.dirname(__file__))
-
-# Get the long description from the README file
-with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
-    long_description = f.read()
-
-# Get the required packages
-with open(path.join(here, 'requirements.txt'), encoding='utf-8') as f:
-    install_requires = f.read().splitlines()
-
-# Plotting packages are optional to install
-extras = ["plotting"]
-extras_require = dict()
-for e in extras:
-    req_file = "requirements-{0}.txt".format(e)
-    with open(req_file) as f:
-        extras_require[e] = [line.strip() for line in f]
-
-# Loading version number
-with open(path.join(here, 'dowhy', 'VERSION')) as version_file:
```

Comment:
I wonder how this change interacts with versioneer and the version it picks? Is versioneer compatible with poetry? @petergtz

Reply:
Not sure. Raised the same question in https://github.com/py-why/dowhy/pull/577#discussion_r933053578. I'd assume they're compatible, but it needs to be verified.

Reply:
I'm unsure about versioneer specifically, but we can utilize git tags with a plugin, as noted above.

Reply:
After digging into it a bit more - it will be a similar mechanism, tag-based text-replacement, but we'd likely have to remove versioneer.